### PR TITLE
Styling and format amends

### PR DESCRIPTION
**Pull request by chrishanes**
Some styling, formatting and copy amends to the staff visit pages

### CODE REVIEW

File: app/views/prison/dashboards/show.html.erb [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
       <tr>
-        <th>Prisoner No.</th>
-        <th>Prisoner Name</th>
-        <th>Visit Date</th>
+        <th>Prisoner name</th>
+        <th>Prisoner no.</th>
+        <th>Visit date</th>
         <th>Time slot</th>
-        <th>Lead Visitor</th>
+        <th>Lead visitor</th>
```
Comment (mloughran) [inline app/views/prison/dashboards/show.html.erb]: Are we planning to keep strings in the locale files for the staff interface? /cc @alan @chrishanes

Comment (chrishanes) [inline app/views/prison/dashboards/show.html.erb]: @alan mentioned the staff interface was going to be only in English, but I had started to add copy in the locales file. Shall I add all text into the locale file?

Comment (mloughran) [inline app/views/prison/dashboards/show.html.erb]: Before I joined the project the convention was established to use the locale files for everything, with the justification that it maintains a clean separation between content and code, and that content writers are comfortable editing yaml. Personally I'm not overly concerned either way, but for the sake of consistency I think we should put everything or nothing in locale files, and unless there's a good reason to change let's keep using locale files.

Comment (alan) [inline app/views/prison/dashboards/show.html.erb]: I might have not been clear @chrishanes when discussing locales, @mloughran is right and everything should be on locale files.

**Pull request merged**